conda_build/variants.py

```diff
@@ -626,14 +628,24 @@ def get_package_variants(recipedir_or_metadata, config=None, variants=None):
 def get_vars(variants, loop_only=False):
```

Comment:
@jezdez, the `is_loop_var` function is a refactor of this function which calculates all loop vars into a function that checks if a var is a loop_var.